src/schemas/Banner/index.ts

```diff
@@ -0,0 +1,29 @@
+import { gql } from 'apollo-server-express';
+import { makeExecutableSchema } from 'graphql-tools';
+
+const bannersStr = process.env.BANNERS || '';
+const bannerArray = JSON.parse(bannersStr);
```

Comment:
If the JSON is poorly formed, will this prevent the server from starting?

I just want to confirm that the server will still spin up (and just not show poorly written banners) if we typo the banner data.

Reply:
that's a very good point

I added error handling for 
- no banners
- there's valid json, but it's not an array
- poorly formed json